pygfx/renderers/wgpu/engine/effectpasses.py

```diff
@@ -129,6 +129,23 @@ class FullQuadPass:
 
     wgsl = ""
 
+    # todo: configurable for each render target
+    load_op = wgpu.LoadOp.clear
+
+    # todo: configurable for each render target
+    blend_op = {
```

Comment:
More correct terminology:
```suggestion
    blend_state = {
```